test/jdk/javax/swing/JTabbedPane/bug4613811.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2003, 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 4613811
+ * @summary Scrollable Buttons of JTabbedPane don't
+ *          get enabled or disabled on selecting tab
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual bug4613811
+ */
+
+import java.awt.BorderLayout;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JTabbedPane;
+
+public class bug4613811 {
+    private static final String INSTRUCTIONS = """
+            Select different tabs and check that the scrollable
+            buttons are correctly enabled and disabled.
+
+            When the very first tab (Tab 1) is fully visible,
+            the left arrow button should be disabled.
+
+            If the last tab (Tab 5) is visible, the right arrow
+            button should be disabled.
```

Comment:
on macOS,

When the 1st tab is visible, the left arrow is not at all visible and right arrow is not visible for 5th tab. I guess they are either not rendered or hidden.

Need to modify test instruction as per the platform.

Reply:
Did you miss this?

Reply:
Confirmed the same. May be platform specific.
![image](https://github.com/user-attachments/assets/532cd1cd-93ef-4e4a-b810-de75cf1484ef)
![image](https://github.com/user-attachments/assets/d910e179-cd45-4eca-a00a-3099e66801c3)
![image](https://github.com/user-attachments/assets/249fa7a9-f2d1-4578-bfbb-44e38d40fabd)

Reply:
Test Instructions updated for macOS